lib/streamlit/elements/plotly_chart.py

```diff
@@ -16,17 +16,36 @@
 
 import json
 import urllib.parse
-from typing import cast
+from typing import Any, cast, Dict, List, Set, TYPE_CHECKING, Union
+
+if sys.version_info >= (3, 8):
+    from typing import Final, Literal
+else:
+    from typing_extensions import Final, Literal
+
```

Comment:
@vdonato I changed the imports here to be python version dependent. In the process I discovered that my used of `TypeAlias` actually does require `typing_extensions` to be installed almost unconditionally.

I thought I'd mention it as input to your decision process regarding that dependency.

Reply:
Thanks yet again @harahu!

If we could bug you to make one more batch of changes to these PRs, I think that now that `typing_extensions` is always installed, we might as well remove the branching imports here (I'll make a quick PR removing them in the ~2 places where they already exist, so no need to worry about those).

You mentioned in a comment in another PR that having these branching imports even with `typing_extensions` always installed may be reasonable as it gives us a signal as to when the imports can be made from the main `typing` package, but I feel like this isn't worth the added complexity. I also feel like it'll most likely never be problematic if we entirely forget to change these imports to be from `typings` > `typing_extensions` as we drop support for older versions of Python (and if types get removed entirely from `typing_extensions`, we'll catch it in CI).

I'll go ahead and approve the currently open PRs and can merge them as these last changes are made.